### PR TITLE
Add MobileMessaging ReactNative types defenitions

### PR DESCRIPTION
**Pull request by kostap13**
Types for React Native plugin: https://github.com/infobip/mobile-messaging-react-native-plugin

### CODE REVIEW

File: types/mobile-messaging-react-native/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+    receivedTimestamp?: number | undefined;
+    seenDate?: number | undefined;
+    contentUrl?: string | undefined;
+    seen?: boolean | undefined;
+    geo?: boolean | undefined;
+    originalPayload?: Record<string, string> | undefined; // iOS only
+    vibrate?: boolean | undefined; // Android only
+    icon?: string | undefined; // Android only
+    category?: string | undefined; // Android only
+    chat?: string | undefined;
```
Comment (tjuric) [inline types/mobile-messaging-react-native/index.d.ts:160]: boolean

File: types/mobile-messaging-react-native/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+export interface GeoCenter {
+    lat: number;
+    lon: number;
+}
+
+export interface Message {
+    messageId: string;
+    title?: string | undefined;
+    body?: string | undefined;
```
Comment (tjuric) [inline types/mobile-messaging-react-native/index.d.ts:146]: should be mandatory

Comment (kostap13) [reply types/mobile-messaging-react-native/index.d.ts]: fixed

File: types/mobile-messaging-react-native/index.d.ts (hunk 1 above)
Comment (kostap13) [reply types/mobile-messaging-react-native/index.d.ts]: Fixed

**Pull request closed without merging**